src/client/components/leaderboard/leaderboard.js

```diff
@@ -26,7 +37,10 @@ class Leaderboard extends React.Component {
               {this.props.scores.map((val,idx) => 
                 <tr key={idx}>
                   <td>{idx}</td>
-                  <td>{this.props.names[idx]} <strong>{(parseInt(this.props.playerID) === idx) ? '(you)' : ""}</strong></td>
+                  <td>{this.props.names[idx]} <strong> {
```

Comment:
Remark: This could be refactored to
```js
<td>
  { this.props.names[idx] } { parseInt(this.props.playerID) === idx && <strong>(you)</strong> }
</td>
```